app/src/main/java/com/movies/common/extensions/ViewExtension.kt

```diff
@@ -40,6 +40,10 @@ fun TextView.changeText(@StringRes textRes: Int) {
     text = context.getString(textRes)
 }
 
+fun View.changeBackground(@DrawableRes drawableRes: Int){
+    background = ContextCompat.getDrawable(context, drawableRes)
+}
+
```

Comment:
technically it is better to change word "change" to "set" because in those extensions you don't delete old value and replace with new one

Reply:
there already is a function setBackgroundColour in java which shadows my extension if I change the name. That's why I named it changeBackgroundColour